bpf/lib/nodeport.h

```diff
@@ -2895,7 +2895,7 @@ static __always_inline int nodeport_lb4(struct __ctx_buff *ctx,
 		/* lookup with SCOPE_FORWARD: */
 		__ipv4_ct_tuple_reverse(&tuple);
 
-		ret = ct_lazy_lookup4(get_ct_map4(&tuple), &tuple, ctx, l4_off, has_l4_header,
+		ret = ct_lazy_lookup4(get_ct_map4(&tuple), &tuple, ctx, ip4, l4_off, has_l4_header,
```

Comment:
Not sure if you've seen the verifier reject in CI. But it's almost certainly complaining about this occurence, as the rewrites in `lb4_local() ` will invalidate the pointer.

Reply:
I just randomly poked around in the red CI results, saw the error message in https://github.com/cilium/cilium/actions/runs/6696314287/job/18194555181?pr=25340 and had a look at the agent log in the sysdump.